.changeset/shy-pens-play.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crackle/cli': minor
+'@crackle/core': minor
```

Comment:
No new API, so I feel like these should be patches.
```suggestion
'@crackle/cli': patch
'@crackle/core': patch
```

Reply:
I thought this warranted a "breaking" change (i.e. `minor` when <1.0) because the underlying implementation has changed significantly.